src/Sensors.cpp

```diff
@@ -139,7 +139,8 @@ void Sensors::setCO2RecalibrationFactor(int ppmValue)
 }
 
 void Sensors::setCO2AltitudeOffset(float altitude){
-    if (getPmDeviceSelected().equals("SCD30")) {
+    if (true) {
```

Comment:
it is unacceptable in any scenery. If the sensor detection and initialization pass ok, it will be true. We don' t need it for the test or example